tests/framework/installer/ceph_helm_installer.go

```diff
@@ -77,7 +77,7 @@ func (h *CephInstaller) CreateRookCephClusterViaHelm(values map[string]interface
 	values["configOverride"] = clusterCustomSettings
 	values["toolbox"] = map[string]interface{}{
 		"enabled": true,
-		"image":   "rook/ceph:master",
+		"image":   "rook/ceph:" + LocalBuildTag,
```

Comment:
it's good to use the `LocalBuildTag` alternatively we could just tag the `LocalBuildTag` build to `rook/ceph:master` locally.